### DISCUSSION TITLE
Q - Does YT-Spammer-Purge also ban emojis between domain names? And does it ignore upper & lowercase?

### DISCUSSION THREAD

**Discussion by nsde** [ThioJoe/YT-Spammer-Purge#1047] [Q&A] [answered]
I've submitted some spam domains, and I'm wondering whether they are going to be detected properly or not.

The thing is, a lot of spammers write their domains like this:
> EXAMPLE🟡com

Will they be detected?
Thanks

**Comment by ThioJoe** [comment, marked as answer]
yes it should
